tests/parsers/test_pwscf.py

```diff
@@ -10,7 +10,7 @@ class TestPWSCFParser(unittest.TestCase):
     def get_parser(self,name):
         '''Get a PwscfParser for a certain test'''
         unpack_example(os.path.join('examples', 'pwscf', name+'.tar.gz'))
-        return PwscfParser(name)
+        return PwscfParser(name, files=None)
```

Comment:
Why set it directly?  It actually re-scopes the test from "default value of `files`" to "`files=None`", which introduces a blind spot.